### ISSUE TITLE
Doesn't support macros

### ISSUE THREAD

**Issue by torkleyy** [eqrion/cbindgen#165]
Macros are quite useful for generating FFI code. I'm not sure how hard it would be to expand macro invocations.

**Comment by RReverser** [comment]
It actually does expand macros under "expand" option. Did you enable it?

**Comment by eqrion** [comment]
@torkleyy Are you using cbindgen from inside a build.rs script? If so then it's probably #121.

**Comment by torkleyy** [comment]
Yes I was, I guess that's the issue. Thanks! I'll close it, in case there's something else that doesn't work I'll reopen.
